### ISSUE TITLE
The HMD in the steamVR window stays grey.

### ISSUE THREAD

**Issue by Lightsteed** [polygraphene/ALVR#7]
The picture comes through PERFECTLY, tracking is incredible, however steamVR is showing the hmd as grey - and i think this is causing me to not be able to use openVR input emulator to change the hmd offsets and also remap my vive controllers in order to get vive controls aswell as positional tracking for the HMD via a controller/vive puck.

Any ideas?

**Comment by polygraphene** [comment]
Thanks!
The gray icon was just my wrong.
It is because of wrong path specification. 8ed8be54052c1d675022b8d61e025f0e21fac884
And now fixed.

I don't know why OpenVR-InputEmulator is not working on ALVR HMD.
I will try it.

**Comment by polygraphene** [comment]
I've tried to fix (b79e700d2ebfa7e62bc06a9cc736257358fed485).
But still crashes... We need more investigation.
Please be patient.

**Comment by Lightsteed** [comment]
Hi, i managed to fix the issue causing openvr input emulator to crash when using with alvr. I found the fix here - https://github.com/matzman666/OpenVR-InputEmulator/issues/10. I successfully managed to remap the tracking from a vive controller to the oculus go hmd. As you mentioned though, the video is not nearly as smooth, but it is aparantly still smooth when using kinect for positional tracking, so it may be a matter of only using the vive tracker for position and still using the go imu data for rotation, not sure joe that would work tho
